app.js

```diff
@@ -138,22 +138,26 @@ function getId(id) {
   }
```

Comment:
### Optional and Curious
Why I'm i not seeing your profile behind the commits?
![Screenshot from 2022-09-08 12-16-58](https://user-images.githubusercontent.com/46388113/189121998-db43290d-0533-47eb-8c90-a80fb2b63ea3.png)